### PR TITLE
login functionality WORKING

### PR DESCRIPTION
**Pull request by cgf5033**
ignore styling on NAV / sidebar (I added a log in button so make sure that is included)

### CODE REVIEW

**Comment by 7socks** [comment]
Please remove package-lock from this PR. That should simplify the conflicts enough to be fixed directly on github.

File: client/src/appState/reducers/user.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
-
-
-
-
 function reducer(state, action) {
 
   const { type, payload } = action
-  const { dev } = state
+  const dev = JSON.parse(JSON.stringify(state.dev));
   // console.log({ type, payload })
```
Comment (7socks) [inline client/src/appState/reducers/user.js:5]: Curious why this change was made?

**Review by Doobss — APPROVED** [review]
Is it login or log in? lol Good stuff dude.

File: client/src/appState/reducers/user.js (hunk 1 above)
Comment (cgf5033) [reply client/src/appState/reducers/user.js:5]: Ian did this to de-reference state.dev

**Review by 7socks — APPROVED** [review]
